index.html

```diff
@@ -14,13 +14,13 @@
 <body>
   <div class="wrapper">
     <header class="tool-bar">
-      <a href="#" class="logo"><h2>John Doe</h2></a>
+      <a href="#" class="logo"><h1>John Doe</h1></a>
```

Comment:
- [ ] Since you have already wrapped your logo with an `a` tag, then why don't you remove the `h1` tag you have brought here because `h1` is usually used for the page title, not for the logo? :smile: